crates/cloud/src/client.rs

```diff
@@ -62,9 +62,10 @@ impl Client {
         let configuration = Configuration {
             base_path,
             user_agent: Some(format!(
-                "{}/{}",
+                "{}/{} spin/{}",
                 env!("CARGO_PKG_NAME"),
-                env!("CARGO_PKG_VERSION")
+                env!("CARGO_PKG_VERSION"),
+                std::env::var("SPIN_VERSION").unwrap_or_default()
```

Comment:
I would maybe do this to help out UA parsers that expect something after the `/`:

```suggestion
                std::env::var("SPIN_VERSION").unwrap_or_else(|| "0".to_string())
```